packages/sanity/src/_internal/cli/actions/validation/reporters/prettyReporter/util.ts

```diff
@@ -33,7 +33,7 @@ const percentageFormatter = new Intl.NumberFormat('en-US', {
 /**
  * Given a decimal, this will return that number formatted as a percentage
  */
-export const percent = (value: number): string => percentageFormatter.format(Math.max(value, 1))
+export const percent = (value: number): string => percentageFormatter.format(Math.min(value, 1))
```

Comment:
This was a bug I introduced last release. Currently the command always says 100% lol